l1-contracts/contracts/bridgehub/MessageRoot.sol

```diff
@@ -167,4 +168,16 @@ contract MessageRoot is IMessageRoot, ReentrancyGuard {
 
         emit AddedChain(_chainId, cachedChainCount);
     }
+
+    /**
+     * @dev Returns merkle path in `sharedTree` for a certain chain.
+     * @param _chainId Id of the chain to get merkle path for.
+     */
+    function getMerklePathForChain(uint256 _chainId) public view returns (bytes32[] memory) {
```

Comment:
not a big deal. but it would indicate that it is not used locally
```suggestion
    function getMerklePathForChain(uint256 _chainId) external view returns (bytes32[] memory) {
```